src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -45,6 +44,38 @@ See the
 [`TestJaegerExporter.cs`](../../examples/Console/TestJaegerExporter.cs)
 for an example of how to use the exporter.
 
+## Resources and Process Tags
+
+The attributes of a [Resource](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md)
```

Comment:
I think its enough to state that Resource attributes are exported as ProcessTags to jaeger, and that service.name is used as processname in jaeger.

The example code - I'd say move it to the console example testjaeger.cs

Reply:
Reworded the entire paragraph according to @CodeBlanch 's suggestions below. Will move the example code next.